### ISSUE TITLE
A task is created during scrolling

### ISSUE THREAD

**Issue by roman-yukhno-axon** [ANovokmet/svelte-gantt#233]
When I try to scroll by dragging the scrollbar, it causes a task to be created. This reproduced in Firefox. How can I fix this?

![Screenshot from 2024-07-14 19-58-57](https://github.com/user-attachments/assets/7add6e93-11d2-4f9b-a18d-d57e67aa1a7f)

**Comment by ANovokmet** [comment]
I am not sure how to fix this. First idea is to disable creating tasks when dragging near the edges of the gantt content. Nevertheless this is an issue with the create tasks feature.

**Comment by roman-yukhno-axon** [comment]
Yes, that's exactly how I fixed it. I thought maybe there was a better solution. I'll leave this here then in case anyone encounters the same problem

```
  private addScrollBehaviour() {
    this.timelineBody = document.querySelector('.sg-timeline-body');
    if (this.timelineBody) {
      document.addEventListener('mousemove', this.mouseMoveHandler.bind(this));
    }
  }
  
  private mouseMoveHandler(event) {
    this.gantt.$set({ enableCreateTask: !this.isCursorNearEdges(event.clientX, event.clientY) });
  }
  
  private isCursorNearEdges(mouseX, mouseY) {
    const rect = this.timelineBody.getBoundingClientRect();
    const buffer = 20; // px
    const nearBottomEdge = mouseY <= rect.bottom && mouseY >= rect.bottom - buffer;
    const nearRightEdge = mouseX <= rect.right && mouseX >= rect.right - buffer;
    return nearBottomEdge || nearRightEdge;
  }
  ```

**Comment by ANovokmet** [comment]
Thank you! I will incorporate this into the library

**Comment by ANovokmet** [comment]
Fixed in v4.4.2
